### ISSUE TITLE
Going to Reports > Docket Report > Search for docket doesn't work

### ISSUE THREAD

**Issue by mlissner** [freelawproject/recap#249]
Just got a bug reported by a user. 

We've optimized the workflow of 

> Login > Query > Docket Report, 

but 

> Login > Reports > Docket Report 

Neither uploads nor provides a notice to the user that something might be free.

Steps to Reproduce:

1. Log into PACER (https://ecf.pawb.uscourts.gov)
2. Click the Reports button.
3. Select Docket Report
4. Put in a docket number (18-21428-TPA)
5. Hit "Find this Case" button
6. Run the report to buy the docket.

Expected results:

1. After step five above, you should see a notice saying whether we have that docket.
2. After step six above, when you see the docket itself, it should get uploaded to us (it's not presently).

**Comment by s-zeid** [comment]
I just ran into this issue with 2:17-cv-00455 in the Eastern District of Virginia.  Following the steps to reproduce (with the appropriate domain name and docket number) reproduces the issue, but using <https://ecf.vaed.uscourts.gov/cgi-bin/DktRpt.pl?372143> (PACER case ID in the query string) to access the case gives the expected results.

To get the PACER case ID, I had to view the console for the RECAP background page to find output such as:

```
RECAP: Got case number '372143' for pacer_doc_id: '18908914387' recap.js:20:17
RECAP: Attempting PDF upload to RECAP Archive with details: pacer_court: vaed, pacer_case_id: 372143, pacer_doc_id: 18908914387, document_number: 21, attachment_number: 0.
```

The issue can be reproduced even after the docket is uploaded using the query string URL.
